migrations/202011301208123-update-data-sellers-add-blockchain-id.js

```diff
@@ -0,0 +1,34 @@
+const global_config = require('../config/config');
+
+if (!process.env.NODE_ENV) {
+    // Environment not set. Use the production.
+    process.env.NODE_ENV = 'testnet';
+}
+const environmentConfig = global_config[process.env.NODE_ENV];
+const blockchain_id = environmentConfig.blockchain.implementations[0].network_id;
```

Comment:
This means that we expect that in new configuration old network_id is in first place in array?

Reply:
Yes, I think so. I'll add this as a comment in the migration story so we don't overlook this detail.